test/test_mps.py

```diff
@@ -1099,6 +1099,52 @@ def test_norm(self):
         res_cpu = torch.norm(d_cpu[0, :, :]), torch.norm(d_cpu[1, :, :])
         self.assertEqual(res, res_cpu)
 
+        e = torch.tensor([[8.9453,  4.0859,  0.1230,  2.1367, -5.0000],
+                  [ 7.2773, -4.6953, -3.5586,  8.2812, -0.8789],
+                  [ 0.7119, -1.4854,  6.8633, -7.9805, -3.6562], 
+                  [-1.0195, -7.2695, -0.0264, -3.5078, -0.2900],
+                  [ 8.7656,  5.8984, -2.3125, -0.0352,  5.2812]],
+                  device='mps', dtype=torch.float16)
+        res = torch.norm(e, float(0.5))
+        # cuda result
```

Comment:
Ziyu is using a separate .yaml file for coda results. Please see if it makes sense to move this data in that file.

Reply:
Agreed, @shuhand0 can you move this to the yaml file.